### ISSUE TITLE
META issue for current IOS problems

### ISSUE THREAD

**Issue by escamoteur** [flutter/flutter#41115]
Currently I observed several problem concerning ios, especially ios 13 and  iPhone XR
The problem is that these users are caught between stable and latest master because some problems occur on stable, others on master

https://github.com/flutter/flutter/issues/40982
https://github.com/flutter/flutter/issues/41085
https://github.com/flutter/flutter/issues/37699
https://github.com/flutter/flutter/issues/41104
https://github.com/flutter/flutter/issues/41092
https://github.com/flutter/flutter/issues/41046
https://github.com/flutter/flutter/issues/41033
https://github.com/flutter/flutter/issues/40998
https://github.com/flutter/flutter/issues/41073

**Comment by eseidelGoogle** [comment]
@christopherfujino @HansMuller for triage.

**Comment by christopherfujino** [comment]
I went through the 6 of these that seemed tool related, hopefully @HansMuller can look at the others.

**Comment by DavBfr** [comment]
Another one here: #41148

**Comment by HansMuller** [comment]
I've looked at the issues as well. @goderbauer  has been looking at  #41073. Maybe @mklim could look at the iOS 13 maps issue (#40982), it appears to be a problem with the maps plugin Marker class. The image picker regression, #41046, needs attention from someone familiar with that plugin. Maybe @cyanglaz.

**Comment by escamoteur** [comment]
Hi, https://github.com/flutter/flutter/issues/37699 seems still not be fixed. A lot of new comments this morning.

**Comment by stuartmorgan** [comment]
Summarizing very briefly from some discussion in Discord: it seems that this bug has as somewhat unclear meaning at this point. Part of the list is "things that are different on iOS between stable and master" and part is "iOS 13 regressions".

My personally recommendation would be to pull out the latter (iOS 13 regressions) into a specific separate meta-bug or label, and then decide whether what's left here has after moving those has a clearly-defined meaning, and if so clarify that.

**Comment by kazenokage** [comment]
#41101 should be added to the list as well as it is affecting all apps using default fonts on iOS 13.

**Comment by jmagman** [comment]
+1 to @stuartmorgan's point.  Can we close this in favor of either labels or a better-defined mega-bug?

**Comment by escamoteur** [comment]
Yep, I made this before the latest hotfix when we got so many issues thst seemed related to iOS 13.
I made already a new label 'OS Version specific'

**Comment by jmagman** [comment]
Closing then.
